src/api/currency/content-types/currency/schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "kind": "collectionType",
+  "collectionName": "currencies",
+  "info": {
+    "singularName": "currency",
+    "pluralName": "currencies",
+    "displayName": "Financial.CurrencyConversion"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "year": {
+      "type": "string",
+      "default": "2020",
+      "required": true,
+      "maxLength": 4
+    },
+    "month": {
+      "type": "integer",
+      "default": 1,
+      "required": true,
+      "max": 12,
+      "min": 1
+    },
+    "currency": {
+      "type": "enumeration",
+      "enum": [
+        "USD",
+        "GBP",
+        "EUR",
+        "LBP",
+        "LTL",
+        "RSD",
+        "BAM"
```

Comment:
**thought:** I think it does make the most sense to have this as an enum, and you have added all of the currencies that were currently in the spreadsheet, however it is likely in the future that there will be more currency types to be considered. I don't think there's anything to be done here right now, but maybe we should add a task to fill out the currencies list?

Reply:
@TravisAlmey agree that we can move forward with this as-is for now.  The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) standard describes the 3-digit code for currencies, and someone has already made an [open-source country-input plugin](https://market.strapi.io/plugins/strapi-plugin-country-select) that maps to ISO codes which would provide a solid example to follow.

@TravisAlmey @ll-zerr can one of you make an issue for this so we don't forget Travis' idea and can track further discussion there?